components/alert_banners/README.md

```diff
@@ -6,57 +6,216 @@ author: orinevares
 ---
 
 ![Status](https://img.shields.io/badge/Recommended-Draft-orange.svg)
-> Last Updated: April 3, 2019
+> Last Updated: April 15, 2019
 
 # Alert Banners
-
-Alert banners notify users of important information or changes on a page. Typically, they appear after at the top of a page.
+Alert banners notify users of important information or changes on a page. Typically, they appear at the top of a page.
 
 ## Example
-
-<component-preview path="components/alert_banners/sample.html" height="100px" width="800px"> </component-preview>
+<component-preview path="components/alert_banners/sample.html" height="688px" width="800px"> </component-preview>
 
 ## Use This For
 **Validation**
 * Alerting users they just did something successful or if it needs to be corrected
 
 **Notification**
```

Comment:
Validation and Notification sort of overlap - In validation it says something success or needs correcting, but errors under notification are considered a form of validation. Would it be easier to change the guidance and say.
Use This For
Telling users  important and typically time sensitive information. This includes:
- Success messages
- Error messages
- Warnings
- Important pieces of information